pkg/helpers/queuekey.go

```diff
@@ -119,6 +128,21 @@ func ClusterManagerDeploymentQueueKeyFunc(clusterManagerLister operatorlister.Cl
 	}
 }
 
+func ClusterManagerSecretQueueKeyFunc(clusterManagerLister operatorlister.ClusterManagerLister) factory.ObjectQueueKeyFunc {
+	return func(obj runtime.Object) string {
+		clustermanagers, err := clusterManagerLister.List(labels.Everything())
+		if err != nil {
+			return ""
+		}
+
+		for _, clustermanager := range clustermanagers {
+			return clustermanager.Name
```

Comment:
I am not sure I understand this. Why it only return the first one?

Reply:
It would be a mistake, `ClusterManagerSecretQueueKeyFunc` is copied from `ClusterManagerConfigmapQueueKeyFunc` but now after a second thought, these functions doesn't work within a multi-cluster-manabger context.
Fixed.